### ISSUE TITLE
conform.input hidden issue

### ISSUE THREAD

**Issue by JIAHONGZHANG** [edmundhung/conform#186]
Hi Ed,

When I was using:

<input
	{...conform.input(suburb, { hidden: true })}
	className="border"
/>

Hidden props not working.
And there is a warning:

React does not recognize the `hiddenStyle` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `hiddenstyle` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

**Comment by JIAHONGZHANG** [comment]
Hidden prop is working once I add ariaAttributes to true.
But hiddenStyle is still showing in input tag

**Comment by jansedlon** [comment]
The following patch fixes it for now(pnpm patch)

```diff
diff --git a/helpers.mjs b/helpers.mjs
index f28204544b7a938d3ed9ea01c35dca76cf4d74f6..933951f9653a6bfd512ca9b67538ac99c081b9fd 100644
--- a/helpers.mjs
+++ b/helpers.mjs
@@ -24,14 +24,14 @@ function getFormControlProps(config, options) {
   if (config.initialError && Object.entries(config.initialError).length > 0) {
     props.autoFocus = true;
   }
-  return options !== null && options !== void 0 && options.hidden ? _objectSpread2(_objectSpread2({}, props), hiddenProps) : props;
+  return options !== null && options !== void 0 && options.hidden ? {...props, ...hiddenProps, style: {...props.style, ...hiddenProps.style}} : props;
 }
 var hiddenProps = {
   /**
    * Style to make the input element visually hidden
    * Based on the `sr-only` class from tailwindcss
    */
-  hiddenStyle: {
+  style: {
     position: 'absolute',
     width: '1px',
     height: '1px',
```

**Comment by JIAHONGZHANG** [comment]
Thanks, jansedlon! It works.

**Comment by edmundhung** [comment]
This should now be fixed on [v0.7.1](https://github.com/edmundhung/conform/releases/tag/v0.7.1). Thanks @JIAHONGZHANG for reporting the bug and @jansedlon for the patch.
